slack-lint-annotations/src/main/kotlin/slack/lint/annotations/AllowUnitResponse.kt

```diff
@@ -0,0 +1,11 @@
+package slack.lint.annotations
+
+/**
+ * Annotation to allow suspend Retrofit functions to return Unit.
+ * 
+ * When applied to a suspend Retrofit function, this annotation permits the function 
+ * to have a return type of Unit, which otherwise will be flagged as an issue.
+ */
+@Target(AnnotationTarget.FUNCTION)
+@Retention(AnnotationRetention.BINARY)
```

Comment:
Marked as Binary retention as we don't need this at runtime.

Reply:
Can it just be SOURCE?